src/main/scala/ai/privado/exporter/JSONExporter.scala

```diff
@@ -64,6 +63,7 @@ import ExecutionContext.Implicits.global
 import duration._
 import scala.language.postfixOps
 import scala.util.{Failure, Success}
+import io.shiftleft.semanticcpg.language._
```

Comment:
remove the unused import

Reply:
This is used for IDE to resolve some implicit function. Not related to any compile time or runtime time dependency.